### PR TITLE
Documenting web desktop variables for the deployer's benefit

### PR DESCRIPTION
**Pull request by c-mart**
@julianpistorius @lenards Am I close?

### CODE REVIEW

File: dist_files/variables.yml.dist [hunk 1]
```diff
@@ -172,13 +172,18 @@ AUTH_MOCK_USER: "atmosphere_user"
 ###
 # Web Desktop
 ###
+# Salts for signing/verifying authentication tokens and creading/decoding browser fingerprints
```
Comment (lenards) [inline dist_files/variables.yml.dist]: I think there is a typo here: `s/creading/creating/` ? 

✅ Comment is correct

File: dist_files/variables.yml.dist [hunk 2]
```diff
@@ -172,13 +172,18 @@ AUTH_MOCK_USER: "atmosphere_user"
...
 # WEB_DESKTOP_SIGNING_SALT: ""
 # WEB_DESKTOP_FP_SALT: ""
+# Shared secrets for signing/verifying authentication tokens and creading/decoding browser fingerprints
```
Comment (lenards) [inline dist_files/variables.yml.dist]: Same potential typo. 

✅ Comment is correct

File: dist_files/variables.yml.dist [hunk 3]
```diff
@@ -172,13 +172,18 @@ AUTH_MOCK_USER: "atmosphere_user"
...
 # WEB_DESKTOP_SIGNING_SECRET_KEY: ""
 # WEB_DESKTOP_FP_SECRET_KEY: ""
-# WEB_DESKTOP_INCLUDE_LINK: False
+# Troposphere feature flag to enable Web Desktop?
+# WEB_DESKTOP_INCLUDE_LINK: False  
+# Should be set to "https://my-novnc-gateway-server.com/vncws"
```
Comment (lenards) [inline dist_files/variables.yml.dist]: ✅ Correct

The hope of using `_URL` suffix is that help indicate that more than the domain is needed.

File: dist_files/variables.yml.dist [hunk 4]
```diff
@@ -172,13 +172,18 @@ AUTH_MOCK_USER: "atmosphere_user"
...
 # WEB_DESKTOP_PROXY_URL: ""
-# WEB_DESKTOP_PROXY_DOMAIN: ""
+# Should be set to ".my-novnc-gateway-server.com", allows Troposphere to set a cross-domain cookie readable by your NoVNC gateway server
+# WEB_DESKTOP_PROXY_DOMAIN: ""
```
Comment (lenards) [inline dist_files/variables.yml.dist]: ✅ Correct

A better name for this might be to include "COOKIE". I think this has potential to be quite confusing.

Comment (lenards) [inline dist_files/variables.yml.dist]: (and I accept that I picked that variable name, sorry)

**Comment by julianpistorius** [comment]
👍

**Comment by lenards** [comment]
I'm good once the typos get corrected. 
(I was going to offer them up as a commit, but I'm not a contributor on the fork)

**Comment by c-mart** [comment]
@lenards fixed, apologies about typos.

**Comment by lenards** [comment]
👍 :shipit:

**Pull request merged**